From 01355d758549bcdecb98f4a8af8626c578db7cfa Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Tue, 6 Aug 2024 18:05:22 +0300
Subject: [PATCH 04/18] arm64: renesas: solidrun: remove can pinmux from common
 dtsi

can bus signal usage is different between SoMs, and possibly different
between carrier boards.
Remove it from common som dtsi to avoid problems when a som or carrier
board redefine can pins.
Since it is not actually used on any previous carrier board it can also
be removed from hummingboard common.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm64/boot/dts/renesas/rz-hummingboard-common.dtsi | 2 --
 arch/arm64/boot/dts/renesas/rz_2l-sr-pinfunction.dtsi   | 5 -----
 2 files changed, 7 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/rz-hummingboard-common.dtsi b/arch/arm64/boot/dts/renesas/rz-hummingboard-common.dtsi
index 17f2115fa..b99957ba5 100644
--- a/arch/arm64/boot/dts/renesas/rz-hummingboard-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/rz-hummingboard-common.dtsi
@@ -89,8 +89,6 @@ ar0521_clk: osc27000_clk {
 };
 
 &canfd {
-	pinctrl-0 = <&can1_pins>;
-	pinctrl-names = "default";
 	status = "disabled";
 	/delete-node/ channel0;
 	channel1 {
diff --git a/arch/arm64/boot/dts/renesas/rz_2l-sr-pinfunction.dtsi b/arch/arm64/boot/dts/renesas/rz_2l-sr-pinfunction.dtsi
index bb5b03c90..b4e886947 100644
--- a/arch/arm64/boot/dts/renesas/rz_2l-sr-pinfunction.dtsi
+++ b/arch/arm64/boot/dts/renesas/rz_2l-sr-pinfunction.dtsi
@@ -181,10 +181,5 @@ ssi1_pins: ssi1 {
 			 <RZG2L_PORT_PINMUX(46, 2, 1)>, /* TXD */
 			 <RZG2L_PORT_PINMUX(46, 3, 1)>; /* RXD */
 	};
-
-    can1_pins: can1 {
-		pinmux = <RZG2L_PORT_PINMUX(40, 0, 3)>, /* TxD */
-			<RZG2L_PORT_PINMUX(40, 1, 3)>; /* RxD */
-	};
 };
 
-- 
2.43.0

